ui/src/components/template/TemplateList.js

```diff
@@ -157,6 +253,28 @@ export default class TemplateList extends React.Component {
                         </tr>
                     </thead>
                     <tbody>{rows}</tbody>
+                    <TitleDiv>Server Templates</TitleDiv>
+                    <thead>
+                        <tr>
+                            <TableHeadStyled align={left}>
+                                TEMPLATE
+                            </TableHeadStyled>
+                            <TableHeadStyled align={left}>INFO</TableHeadStyled>
+                            <TableHeadStyled align={center}>
```

Comment:
if the server template list only contains the templates which haven't be applied to the domain then do we require CurrentVersion column?

Reply:
It's for symmetry with the "existing templates" table above it.

Reply:
ohh in that case it would be better to show 0 or N/A as value instead of leaving it empty

Reply:
Will place N/A

Reply:
Fixed